code/Sitecore.NSubstitute.UnitTests/FakeItemTester.cs

```diff
@@ -90,6 +91,15 @@ public void Constructor_WhenCalled_HasChildrenPropertyInitialized()
             item.Children.Should().NotBeNull();
         }
 
+        [Fact]
+        public void Constructor_WhenCalled_GetChildrenMethodIsSetupInitialized()
+        {
+            Item item = new FakeItem();
+
+            item.GetChildren().Should().NotBeNull();
+            item.GetChildren(ChildListOptions.None).Should().NotBeNull();

```

Comment:
Do we want only specific ChildListOptions to be tested?

What is the expected behavior with other child options?

Reply:
Added possible enum values